gcloud-java-contrib/pom.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.gcloud</groupId>
+  <artifactId>gcloud-java-contrib</artifactId>
+  <packaging>jar</packaging>
+  <name>GCloud Java contrib</name>
+  <description>
+    gcloud-java additional functionality.
```

Comment:
Nit: I think it'd be clearer if we said something along the lines of: "Contains packages that provide higher-level abstraction/functionality for common `gcloud-java` use cases."  If we do change the description here to something more informative, we could use the same description in the README tag line.